.github/workflows/pytest-stripe-disabled.yml

```diff
@@ -0,0 +1,68 @@
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: pytest
```

Comment:
If we are to use this action, let's give it a different name from regular pytest.